libs/blocks/quiz-marquee/quiz-marquee.js

```diff
@@ -68,9 +68,13 @@ export default async function init(el) {
   [...rows].forEach(async (row) => {
     const cols = row.querySelectorAll(':scope > div');
     const isFragRow = cols[0].textContent.trim() === 'nested-fragments';
+    const isList = cols[2]?.textContent?.trim() === 'list' || false;
```

Comment:
Do we need the `|| false` in here? The `=== operator` will return false if the condition is not met.
```suggestion
    const isList = cols[2]?.textContent?.trim() === 'list';
```